content/unsupported-browser.html

```diff
@@ -4,15 +4,28 @@
 robots_tag: noindex
 ---
 
+<script defer src="/assets/js/unsupported-browser.js"></script>
+
 <section id="intro" class="intro-heading">
   <h1 class="mb-3">Unsupported Browser</h1>
   <p class="mb-1">
     Scratch Addons doesn't support the browser you're using.
   </p>
 </section>
 
-<section class="container">
-  <p>Use updated versions of Chrome, Firefox, or other browsers based on them to get started:</p>
+<div class="container">
+
+<section class="mb-5">
+  <p>Your browser is not supported by Scratch Addons.</p>

```

Comment:
Shouldn't you tell the user to "update their browser?" "Your browser is not supported by Scratch Addons." is basically a reiteration of the hero page (see line 12).

For example:

```suggestion
  <p>Please update your browser to use Scratch Addons. We recommend exporting your settings before doing so.</p>
```